package.json

```diff
@@ -38,6 +38,7 @@
         "@babel/preset-env": "^7.16.4",
         "@babel/preset-flow": "^7.16.0",
         "@babel/register": "^7.16.0",
+        "@hyperjump/json-schema": "^0.17.2",
```

Comment:
It seemed to support more recent drafts of the spec and have a better rep over all.

Reply:
That and it was one of the first i landed on that did what I want (I found it via the jsonschema.org site). I could be persuaded to change it. It worked so I stopped looking.

Reply:
I was curious about the speed because I tried to use ajv in the past and it was slow for the schema I had written.  It could also been that my schema was overly complex. 🤷‍♂️